test/webapi/ows/stac/demo-collection.json

```
@@ -7,6 +7,19 @@
       ],
       "type": "application/zarr",
       "href": "http://localhost:8080/s3/datasets/demo.zarr",
+      "xcube:store_kwargs": {
+        "data_store_id": "s3",
+        "root": "datasets",
+        "storage_options": {
+          "anon": true,
+          "client_kwargs": {
+            "endpoint_url": "http://localhost:8080/s3"
+          }
+        }
+      },
+      "xcube:open_kwargs": {
```

```suggestion
      "xcube:open_data_params": {
```

to align it with data store API method `DataStore.open_data()` (-> `open_data`) that takes all the given parameters (-> `params`). And align it API method `DataStore.get_open_data_params_schema()` .